### PR TITLE
Launch inventory update -> Launch Inventory Updates (rename)

### PR DESCRIPTION
**Pull request by MilanPospisil**

### CODE REVIEW

File: frontend/awx/resources/inventories/hooks/useInventoriesSourcesToolbarActions.tsx [hunk 1]
```diff
@@ -92,7 +92,7 @@ export function useInventoriesSourcesToolbarActions(
         type: PageActionType.Button,
         selection: PageActionSelection.None,
         icon: RocketIcon,
-        label: t('Launch inventory update'),
+        label: t('Launch Inventory Updates'),
```
Comment (skochay330) [inline frontend/awx/resources/inventories/hooks/useInventoriesSourcesToolbarActions.tsx:95]:
```suggestion
        label: t('Launch inventory updates'),
```

Comment (MilanPospisil) [reply frontend/awx/resources/inventories/hooks/useInventoriesSourcesToolbarActions.tsx:95]: Are you sure? The tabs names are usually in capital letters...

Comment (skochay330) [reply frontend/awx/resources/inventories/hooks/useInventoriesSourcesToolbarActions.tsx:95]: Looks like the standard across platform UI is lowercase- @tiyiprh please confirm

Comment (tiyiprh) [reply frontend/awx/resources/inventories/hooks/useInventoriesSourcesToolbarActions.tsx:95]: UXD is working on standardizing the casing structure throughout the platform. Currently working on a PR to update the breadcrumbs casing. Tabs should be title case/capital but buttons should be sentence case/lower case.

**Review by jerabekjiri — APPROVED** [review]
:+1:

**Pull request merged**